src/components/DownloadBtn/DownloadBtn.tsx

```diff
@@ -0,0 +1,281 @@
+import { useEffect, useState } from "react"
```

Comment:
the component [DownloadButton](https://github.com/decentraland/ui2/blob/master/src/components/DownloadButton/DownloadButton.tsx) is available in the latest version of `decentraland-ui2`